nodewatcher/modules/analysis/rogue_nodes/tasks.py

```diff
@@ -0,0 +1,81 @@
+import datetime
+
+from django.core.mail import mail_admins
+
+from django.conf import settings
+
+from nodewatcher import celery
+
+from nodewatcher.modules.monitor.http.survey.management.commands.export_survey_data import extract_survey_graph
```

Comment:
I think it would be better if this particular function would be somewhere else, and then both the management command and this task would import it.